app/src/main/res/values/strings.xml

```diff
@@ -1068,4 +1070,14 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_camera_type_dome">Dome</string>
     <string name="quest_camera_type_fixed">Fixed</string>
     <string name="quest_camera_type_panning">Panning</string>
+    <string name="no_search_results">The what now?</string>
```

Comment:
Shouldn't this be a more formal wording?

Reply:
I don't know, why should it?

Reply:
Informal seems fine and even preferable - as long as it is not some slang/insulting

Reply:
I don't know what "The what now" should mean... Maybe the context where it is used will help with understanding.

Reply:
Well I hope it means something like "hä, was jetzt?" but maybe i misunderstood is meaning

Reply:
"hä, was jetzt?" auto-translates to "huh, what now?" As in, "I'm not sure what to do now.". I *think* that's more what you meant.

I would interpret "The what now?" more like "Sorry, what?" or "Are you talking nonsense at me?" Or, in gif form, 

![white guy blinking meme](https://i.imgur.com/Gm3n0F1.gif)

Reply:
Well, that would be fine, ain't it?

Reply:
It is shown when the search word(s) entered do not match any quest types, so the list is empty.

Reply:
It's understandable as a native, but it's very informal/casual, almost cryptic, similar to:
https://www.theguardian.com/lifeandstyle/wordofmouth/2014/mar/25/wackaging-trend-food-packaging-innocent-language

I think you may need to add a hint for translators so they get the right idea.

It also seems out of place compared to the rest of the language used in the app (e.g. connection failures) which is generally a bit less "fun".

Reply:
Although "Too much to do?" below seems similar too.

Reply:
I have no idea what "the what now" should mean either... Something like `No matches` would be shorter, more common, and more understandable. But maybe that is the point (like [borken](https://www.landley.net/history/mirror/jargon.html#borken) for broken)

Or, if we're going for such level of borken/informal, we could go with ["What you say !!"](https://knowyourmeme.com/memes/all-your-base-are-belong-to-us) meme at least :smile:

Reply:
Do you have a suggestion for a slightly less "cryptic" wording, @ native speakers?

> Although "Too much to do?"

That's pretty much German ("Zu viel zu tun?") translated into English. If it doesn't work that well in English, please post a suggestion!

Reply:
Well this is equally applicable, but less fun, but should be more understandable for people who might be using it in a non-native language, or translating it.
```suggestion
    <string name="no_search_results">No results found</string>
```

Reply:
Meh. People don't do their taxes in StreetComplete.